ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -401,11 +402,12 @@ static int
 bb_setOption (void **pdata, int opNum, const char *param)
 {
   bounding_boxes *bdata = *pdata;
-
+  g_message("[BB] opNum: %d, param: %s", opNum, param);
```

Comment:
BTW, is this intended? or debugging messages added in the devel. process?

Reply:
It's for the debugging :sweat_smile:. They're gone!